src/lib/util.test.js

```diff
@@ -0,0 +1,59 @@
+const should = require(`should`);
```

Comment:
I imagined this file would be in a top level tests directory or some directory just for tests. Is that incorrect?